### ISSUE TITLE
`Net::ReadTimeout` circle ci errors

### ISSUE THREAD

**Issue by devanshu-m** [emory-libraries/dlp-curate#799]
Notes:
* Low priority
* `Net::ReadTimeout` errors occur in ci during importer specs.

**Comment by eporter23** [comment]
@rotated8 is this one done (based on slack posts??

**Comment by rotated8** [comment]
No, we're skipping the tests at the moment. I'm maybe halfway through fixing the timeout.
